backend/functions/src/course/routes.ts

```diff
@@ -35,4 +44,51 @@ router.get('/students/:courseId', (req, res) => {
     })
 })
 
+router.get('/semester/current', (_, res) => {
+  getCurrentSemester()
+    .then((data) => res.status(200).send(data))
+    .catch((err) => {
+      logger.error(`Unexpected error getting semester: ${err.message}`)
+      res.status(500).send({ message: err.message })
+    })
+})
+
+router.post('/semester/current', (req, res) => {
+  const { semester } = req.body
+  setCurrentSemester(semester)
+    .then(() => res.status(200).send({ success: true }))
+    .catch((err) => {
+      console.log(err)
+      res.status(400).send({ success: false, err: err.message })
+    })
+})
```

Comment:
just a note, in the future, there will prob be a future task to move all of these. I'm making a mental note now.